### ISSUE TITLE
Create `AdBlockingRecoveryApp` skeleton component and render it in the `ad-blocking-recovery` screen

### ISSUE THREAD

**Issue by kuasha420** [google/site-kit-wp#6964]
## Feature Description

A new `App` component (Similar to `UserInputApp`) should be created for Ad Blocking Recovery on boarding screen. The component should render the main structure of the page, including header, footer, the graphics in the right should match the [Figma](https://www.figma.com/file/9jzqdDjP7pCL8MiYHtxxoJ/Ad-Blocker-Detection-2?node-id=130-2578&t=4rJVnIg0l5lGTq5G-0) design. The [rest of the page](https://www.figma.com/file/9jzqdDjP7pCL8MiYHtxxoJ/Ad-Blocker-Detection-2?node-id=130-2578&t=4rJVnIg0l5lGTq5G-0) will be filled in in a follow up issue once the `Stepper` and any other needed components have been created in separate issue(s). 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

@tofumatt I have two UI observations where there are differences with FIgma.  

These could be expected but would like to check to make sure.

1.  The font size and style of the `Ad Blocking Recovery` title is different.  On my site it is `22px` in `Google Sans Display`. On figma designs it is `20px` in `Google Sans Text` font.  
 
![image](https://github.com/google/site-kit-wp/assets/73545194/0db4181b-218d-4d94-b42d-71b3a2ec81ea)

2. The dimensions of the SVG are also different.  On my test site the SVG is `427.99` x `298.51` but on figma designs it is 
`390px` x `272.01px` 

![image](https://github.com/google/site-kit-wp/assets/73545194/78c75d57-90bc-457d-886c-cac67209a20a)

**Comment by tofumatt** [comment]
@wpdarren Yes, those are both fine, the image will adjust to the viewport size so won't always be exactly as seen in Figma.

And the heading size is consistent with the rest of the headings for pages like that in the plugin, so is all good 🙂

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- When visiting `/wp-admin/admin.php?page=googlesitekit-ad-blocking-recovery` the basic scaffolding of the component is as per the figma designs with the exception of the points mentioned above. 
- The hero graphic does appear on tablet/desktop sizes and not appear on mobile.
- `AdBlockingRecoveryApp` is in storybook and displays.

<details>
 <summary>Screenshots</summary>

![image](https://github.com/google/site-kit-wp/assets/73545194/0db4181b-218d-4d94-b42d-71b3a2ec81ea)
![image](https://github.com/google/site-kit-wp/assets/73545194/78c75d57-90bc-457d-886c-cac67209a20a)
![image](https://github.com/google/site-kit-wp/assets/73545194/25ec1357-db11-488d-8978-4de277757c80)

</details>
